### PR TITLE
fix: autotest configsheet datasource addon search

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
The addon creation interface has been changed, and the query of automatic test configuration sheet needs to be changed

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       The addon creation interface has been changed, and the query of automatic test configuration sheet needs to be changed       |
| 🇨🇳 中文    |        addon 创建接口有变更，需要变更下自动化测试配置单的查询      |

### CODE REVIEW

File: modules/orchestrator/endpoints/addon.go [hunk 1]
```diff
@@ -306,7 +306,7 @@ func (e *Endpoints) ListAddon(ctx context.Context, r *http.Request, vars map[str
 
 			var find = false
 			for _, displayName := range displayNames {
-				if v.AddonDisplayName == displayName {
+				if v.AddonName == strings.ToLower(displayName) {
```
[the remark this answers is not in the document]
Comment (kakj-go) [reply modules/orchestrator/endpoints/addon.go:309]: The addon creation interface has changed. Previously, addondisplayname was just a type, but now it has the same value as name

**Review by sfwn — APPROVED** [review]

**Pull request merged**